Packs/EWS/Integrations/EWSO365/EWSO365_test.py

```diff
@@ -360,13 +360,14 @@ def test_handle_html(mocker, html_input, expected_output):
     assert handle_html(html_input) == expected_output
 
 
-@freeze_time('2021-05-23 13:18:14.901293+00:00')
-@pytest.mark.parametrize('since_datetime, filter_arg, expected_result',
-                         [('', 'last_modified_time__gte', EWSDateTime.from_string('2021-05-23 13:08:14.901293+00:00')),
+@pytest.mark.parametrize('since_datetime, filter_arg, expected_result, parse_result',
+                         [('', 'last_modified_time__gte', EWSDateTime.from_string('2021-05-23 13:18:14.901293+00:00'),
+                           datetime.datetime(year=2021, month=0o5, day=23,
+                                             hour=13, minute=18, second=14, microsecond=901293)),
```

Comment:
better to revert the change here and use freeze_time like before - please check with Itay when it's supposed to be merged and accordingly we will decide what to do